docs/rules/consistent-test-it.md

```diff
@@ -1,14 +1,26 @@
 # Have control over `test` and `it` usages (`consistent-test-it`)
 
+<!-- RULE_NOTICE -- Generated by `yarn tools:regenerate-docs` -->
+
+<!-- prettier-ignore -->
+💼 This rule is enabled in the following [configs](https://github.com/jest-community/eslint-plugin-jest#shareable-configurations): `all`.
+
+<!-- prettier-ignore -->
+🔧 This rule is automatically fixable using the `--fix` [option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix) on the command line.
+
+<!-- /RULE_NOTICE -->
```

Comment:
I'd really love it if we could somehow not need these markers all together, but I can't think of a good way to match that - still, since we're wanting these to always be at top and everything until this line is generated I think we can just use a "end rule notices" marker (I'd like that name too, because its consistent with the format of the markers in the readme)

I think it would also make sense to have the tool add this if it can't find the marker, which would make it a bit smoother if you're making a new rule.

Reply:
I switched to using an "end rule header" marker only. And the tool will update the header or create it if it doesn't exist.